apps/examples/log_dump/log_dump_main.c

```diff
@@ -43,6 +44,9 @@ int log_dump_main(int argc, char *argv[])
 #endif
 {
 	int ret = 1;
+#ifdef CONFIG_BUILD_FLAT
```

Comment:
Could you let me know why this conditional is necessary?

Reply:
Please implement size api in procfs. Then this is not required.